worlds/v6/__init__.py

```diff
@@ -23,46 +23,62 @@ class V6Web(WebWorld):
 
 
 class V6World(World):
-    """ 
+    """
      VVVVVV is a platform game all about exploring one simple mechanical idea - what if you reversed gravity instead of jumping?
-    """ #Lifted from Store Page
-
-    game: str = "VVVVVV"
-    topology_present = False
-    web = V6Web()
+    """
 
+    game = "VVVVVV"
+    topology_present = True
     item_name_to_id = item_table
     location_name_to_id = location_table
-
+    option_definitions = v6_options
     data_version = 1
-
+    web = V6Web()
     area_connections: typing.Dict[int, int]
-    area_cost_map: typing.Dict[int,int]
-
-    music_map: typing.Dict[int,int]
-
-    option_definitions = v6_options
+    area_cost_map: typing.Dict[int, int]
+    music_map: typing.Dict[int, int]
 
     def create_regions(self):
         create_regions(self.multiworld, self.player)
 
     def set_rules(self):
-        self.area_connections = {}
-        self.area_cost_map = {}
-        set_rules(self.multiworld, self.player, self.area_connections, self.area_cost_map)
+        set_rules(self.multiworld, self.player)
+
+    def create_item(self, name: str, progression: bool = False) -> V6Item:
+        return V6Item(
+            name,
+            ItemClassification.progression if progression else ItemClassification.filler,
+            item_table[name],
+            self.player)
+
+    def generate_early(self):
+        self.area_connections = {0: 0, 1: 1, 2: 2, 3: 3, 4: 4}
+        self.area_cost_map = {0: 0, 1: 1, 2: 2, 3: 3, 4: 4}
```

Comment:
`{i : i for i in range(5)}`

Reply:
Was a small optimization, but I can change it to that if you prefer.

Reply:
With a game as small as this i'd prefer it that way, tbh. Probably gets cached anyway